### ISSUE TITLE
model 이 리턴하는 output에 hidden_states 와 attentions가 존재하지 않습니다

### ISSUE THREAD

**Issue by wjdghks950** [monologg/KoBERT-nsmc#4]
안녕하세요, KoBERT로 nsmc classification을 하기 위해 여러 실험을 해보던 중 한 가지를 발견했는데, `BertForSequenceClassification`에서 리턴하는 값인 `output` 내에 `(loss), logits, (hidden_states), (attentions)` 이 아닌 `(loss), logits` 만 들어있었습니다. 현재 `from_pretrained("monologg/kobert`를 사용 중인데, 혹시 `monologg/kobert` 버전의 모델은 hidden_states와 attentions는 리턴을 하지 않는건가요?

**Comment by monologg** [comment]
forward를 할 때 `output_attentions=True`, `output_hidden_states=True` 를 인자로 넣으면 가능할 것입니다.

아래 코드를 보길 권장드립니다:)
https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L707-L712

**Comment by wjdghks950** [comment]
감사합니다. 해결되었습니다 👍
